macos.keybindings.json

```diff
@@ -1,1428 +1,1428 @@
 
 [
-      { "key": "escape escape",         "command": "workbench.action.exitZenMode",
```

Comment:
This is a noisy diff because a previous commit #7 didn't follow the whitespace conventions.